### PR TITLE
React issue 2

### PR DESCRIPTION
**Pull request by aman-singh**
Adding name and comment fields to FeedbackForm.

### CODE REVIEW

**Comment by aman-singh** [comment]
[React Issue 2: Add FeedbackStore, and add FeedbackForm](https://trello.com/c/2NNassdL/797-react-issue-2-add-feedbackstore-and-add-feedbackform)

File: app/javascript/image_sharing/components/App.js [hunk 1]
```diff
@@ -2,13 +2,15 @@ import React, { Component } from 'react';
...
 import Footer from './Footer';
+import FeedbackForm from './FeedbackForm';
 
 class App extends Component {
   /* Add Prop Types check*/
   render() {
     return (
       <div>
         <Header title='Tell us what you think' />
+        <FeedbackForm />
```
[the remark this answers is not in the document]
Comment (aman-singh) [reply app/javascript/image_sharing/components/App.js:13]: Thanks! Will keep that in mind for product code dev, will let this be as is.

File: app/javascript/image_sharing/components/FeedbackForm.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+@observer
+class FeedbackForm extends Component {
+  render() {
+    return (
+      <div>
+        <Row>
+          <Col lg={{ size: 4, offset: 4 }}>
+            <form>
+              <div className="form-group">
```
[the remark this answers is not in the document]
Comment (aman-singh) [reply app/javascript/image_sharing/components/FeedbackForm.js:13]: This isn't clear, please let me know which elements your recommending and why are they better.

File: app/javascript/image_sharing/components/FeedbackForm.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+                <input
+                  type="text"
+                  className="form-control"
+                  onChange={(e) => {
+                  this.props.stores.feedbackStore.setName(e.target.value);
+                }}
+                />
+              </div>
+              <div className="form-group">
+                <p id="comment">Comments:</p>
```
[the remark this answers is not in the document]
Comment (aman-singh) [reply app/javascript/image_sharing/components/FeedbackForm.js:24]: what's and? and why?

File: app/javascript/image_sharing/test/stores/FeedbackStore.spec.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import FeedbackStore from '../../stores/FeedbackStore';
+
+describe('FeedbackStore', () => {
+  it('should store values for name and comment', () => {
+    const store = new FeedbackStore();
+    store.setComment('blah blah');
```
[the remark this answers is not in the document]
Comment (aman-singh) [reply app/javascript/image_sharing/test/stores/FeedbackStore.spec.js:8]: Will keep this in mind for real projects, will leave this as it is. Want to spend time working on real projects than making this onboarding project proper.

File: app/javascript/image_sharing/components/FeedbackForm.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sthust) [reply app/javascript/image_sharing/components/FeedbackForm.js:13]: <Form>
     <FormGroup>
         e.g. <Label for="user-name">Name:</Label> are the REACT gears elements from the appfolio UI library that it used for all of APM, this is a way to learn them. https://appfolio.github.io/react-gears/?path=/story/react-gears--introduction

File: app/javascript/image_sharing/components/FeedbackForm.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (sthust) [reply app/javascript/image_sharing/components/FeedbackForm.js:24]: sorry comment got cut off: similar to above:
<FormGroup>
          <Label for="comments">Comments:</Label>
          <Input ...

File: app/javascript/image_sharing/components/FeedbackForm.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (aman-singh) [reply app/javascript/image_sharing/components/FeedbackForm.js:13]: The story doesn't mention using appfolio gear elements so I think I'll skip that for now and learn more about that lib later. Thanks for making me aware of that lib.

**Pull request merged**